build/Build.cs

```diff
@@ -1,38 +1,37 @@
 // ReSharper disable RedundantUsingDirective
-
-using System;
-using System.Linq;
 using Nuke.Common;
-using Nuke.Common.CI;
 using Nuke.Common.Execution;
-using Nuke.Common.Git;
 using Nuke.Common.IO;
 using Nuke.Common.ProjectModel;
-using Nuke.Common.Tooling;
 using Nuke.Common.Tools.DotNet;
 using Nuke.Common.Tools.NuGet;
+using Nuke.Common.Tools.OctoVersion;
 using Nuke.Common.Utilities.Collections;
-using Nuke.OctoVersion;
-using OctoVersion.Core;
-using static Nuke.Common.EnvironmentInfo;
+using Serilog;
 using static Nuke.Common.IO.FileSystemTasks;
-using static Nuke.Common.IO.PathConstruction;
 using static Nuke.Common.Tools.DotNet.DotNetTasks;
 
 [CheckBuildProjectConfigurations]
-[ShutdownDotNetAfterServerBuild]
+[UnsetVisualStudioEnvironmentVariables]
```

Comment:
What does `UnsetVisualStudioEnvironmentVariables` do? First time I've seen it on our Nuke builds.

Reply:
Good question. Not much documentation available for it. (This was taken from [RoslynAnalyzers](https://github.com/OctopusDeploy/RoslynAnalyzers)) 
Will convert to the usual `ShutdownDotNetAfterServerBuild`